backend/app/Hooks/DiscordOAuthHelper.php

```
@@ -283,21 +283,24 @@ public function checkJ4RServerJoins(string $discordId, string $accessToken, Sess
+                } else {
+                    // Emit event for rewards claimed
+                    global $eventManager;
+                    $eventManager->emit(J4REvent::onJ4RRewardsClaimed(), [
+                        'user_uuid' => $session->getInfo(UserColumns::UUID, false),
+                        'username' => $session->getInfo(UserColumns::USERNAME, false),
+                        'coins_earned' => $totalCoinsEarned,
+                        'servers_joined' => $newlyJoinedServers,
+                    ]);
+
+                    $this->app->getLogger()->info('J4R rewards claimed for user: ' . $session->getInfo(UserColumns::USERNAME, false) . ' (+' . $totalCoinsEarned . ' coins)');
+                }
```

PHPMD (2.15.0)

The method checkJ4RServerJoins uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)